packages/opera.rb

```diff
@@ -17,21 +33,23 @@ class Opera < Package
     depends_on 'sommelier'
   end
 
-
-  def self.build
-    Dir.chdir('../') do
-      system "alien -tc opera-stable_#{version}_amd64.deb"
-      system "tar xf opera-stable-#{version}.tgz"
-    end
-  end
-
   def self.install
-    Dir.chdir('../') do
-      # llvm-strip doesn't works with opera
-      ENV['CREW_NOT_STRIP'] = '1'
-      FileUtils.mkdir_p CREW_DEST_PREFIX
-      FileUtils.mv Dir.glob('usr/*'), CREW_DEST_PREFIX
-      FileUtils.ln_s "#{CREW_PREFIX}/bin/opera", "#{CREW_DEST_PREFIX}/bin/x-www-browser" unless File.exist?("#{CREW_PREFIX}/bin/x-www-browser")
+    # llvm-strip doesn't works with opera
+    ENV['CREW_NOT_STRIP'] = '1'
+    
+    if ARCH == 'x86_64' then
+      # since opera put the executable to library, we need to link it to CREW_PREFIX
+      # remove old symlink which link to /usr/lib/x86_64-linux-gnu/opera/opera
+      FileUtils.rm_rf 'bin/opera'
+      FileUtils.ln_s "#{CREW_LIB_PREFIX}/opera/opera", 'bin/opera'
```

Comment:
Would it make sense to move everything from `usr/local/lib/opera` to `#{CREW_DEST_PREFIX}/share/opera`?

Reply:
Will `opt` better?

Reply:
The `opt` location is the alternate way.  I try to keep all packages organized in `share` so we don't have apps in so many different locations.

Reply:
Change made